projects/plugins/jetpack/modules/masterbar/admin-menu/class-atomic-admin-menu.php

```diff
@@ -130,15 +130,17 @@ public function add_users_menu() {
 	 */
 	public function add_plugins_menu() {
 
-		if ( self::CLASSIC_VIEW === $this->get_preferred_view( 'plugins.php' ) ) {
-			return;
-		}
-
 		global $submenu;
 
 		// Calypso plugins screens link.
 		$plugins_slug = 'https://wordpress.com/plugins/' . $this->domain;
 
+		if ( self::CLASSIC_VIEW === $this->get_preferred_view( 'plugins.php' ) ) {
```

Comment:
I think the code here makes sense. The only thing I would add is a comment to explain why this code exists. I think it is not immediately apparent what the purpose is of overwriting a menu item in this case.

Reply:
Sounds good, I will add the comment!